### ISSUE TITLE
Can't compile a Windows app using this library

### ISSUE THREAD

**Issue by ScottArbeit** [dotnet/Microsoft.Maui.Graphics.Controls#105]
Hi! Quick question: I'm trying this library for the first time, and I use Windows, so I was hoping to just compile a Windows app to look at things. When I select Windows in the platform drop-down, I get errors, as shown:

<img width="677" alt="image" src="https://user-images.githubusercontent.com/2406993/151715635-03e4822c-47d4-4aed-82e2-849d0b93d0ef.png">

<img width="809" alt="image" src="https://user-images.githubusercontent.com/2406993/151715661-ad553d8f-d09a-442f-92bb-eb7cd1121f1a.png">

Repro:

1. Open VS2022 (using Version 17.1.0 Preview 4.0, with MAUI Preview 12)
2. Create new .NET MAUI project
3. Add `Microsoft.Maui.Graphics.Controls` NuGet package from nightly feed
4. Add `using` statements and call to `.ConfigureGraphicsControls()`
5. Select Windows from the platform drop-down menu
6. Get errors when trying to compile to Windows

Is this expected right now? I understand that this is experimental, so, if Windows just isn't implemented yet, that's fine. Just want to make sure I'm not being an idiot...

**Comment by dev-thinks** [comment]
yes. not compiling with windows. bump when try adding it from nightly build.

**Comment by EthanHipps** [comment]
@tobiasbaur I encountered a similar error today when I tried adding the library to my project. Preview 13 too, but building for Android instead of Windows. If I comment out `.ConfigureGraphicsControls(DrawableType.Material)` the project builds and works without issue. 

![image](https://user-images.githubusercontent.com/13156894/155758431-deffe473-3795-4375-821f-8b2f6f4045d7.png)

The Exception:
> Could not resolve type with token 010000b2 from typeref (expected class 'Microsoft.Maui.Graphics.Native.NativeGraphicsService' in assembly 'Microsoft.Maui.Graphics, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null')

**Comment by jsuarezruiz** [comment]
This issues with Preview 13 will be fixed with https://github.com/dotnet/Microsoft.Maui.Graphics.Controls/pull/106 (in progress)

**Comment by aliessmael** [comment]
Not compiling, I am using this GitHub code ( not able to find NuGet package ) and I have Visual studio 17.3.0 Preview.

**Comment by KMRH47** [comment]
Also broken here running VS 2022 Version 17.3.0 Preview 3.0

**Comment by MattBDev** [comment]
Will this be fixed anytime soon or is the plan to abandon this library and merge it into .NET MAUI?

**Comment by WongResearch** [comment]
> This issues with Preview 13 will be fixed with #106 (in progress)

Hopefully #106 will make it to the nightly build soon. 

For those using the latest nightly build (294) and must target windows you can try excluding it for now for it to compile for ios and android:

![2022-10-13 at 17 09 02](https://user-images.githubusercontent.com/1698569/195526406-47d09bfb-62fd-4a92-9496-39b97c5dec1d.png)

![2022-10-13 at 17 09 38](https://user-images.githubusercontent.com/1698569/195526542-c5497450-f6a6-4d27-bb12-52833f8ad8d7.png)

**Comment by juliusdeblaaij** [comment]
Is there any progress on this front?

**Comment by mgramley** [comment]
Any update on this bug? I still can't get this library to work with windows
